src/main.ts

```diff
@@ -29,6 +30,19 @@ app.use('/api', userRouter)
 app.use('/api', groupRouter)
 app.use('/api', UserAssignRouter)
 
+//错误级别中间件
```

Comment:
We are supposed to have a middleware for service calling. Log the method, url, request params or body, and response. It's better to have a response time to measure the performance .